static/app/views/issueList/overview.tsx

```diff
@@ -752,11 +754,9 @@ class IssueListOverview extends React.Component<Props, State> {
     this.transitionTo({display});
   };
 
-  onCursorChange = (cursor: string | undefined, _path, query, pageDiff: number) => {
-    const queryPageInt = parseInt(query.page, 10);
-    let nextPage: number | undefined = isNaN(queryPageInt)
-      ? pageDiff
-      : queryPageInt + pageDiff;
+  onCursorChange: CursorHandler = (cursor, _path, _query, delta) => {
+    const queryPageInt = parseInt(this.props.location.query.page, 10);
+    let nextPage: number | undefined = isNaN(queryPageInt) ? delta : queryPageInt + delta;
```

Comment:
The test propagates location changes through the location prop. I had a hard time figuring out how to get the react router context to update.

I wouldn't usually change implementation to accommodate for tests, but in this case I'm OK with it since it's such a pita test file.